examples/images/index.js

```diff
@@ -37,7 +37,14 @@ const Image = styled('img')`
  */
 
 function isImage(url) {
-  return !!imageExtensions.find(url.endsWith)
+  return !!imageExtensions.find(
+    element =>
+      element ===
```

Comment:
Hi @eroepken, what do you think about wrapping this in a function, like `getExtension(url)`?

Reply:
I can definitely do that.